packages/fiber/tests/web/canvas.test.tsx

```diff
@@ -1,10 +1,13 @@
 jest.mock('scheduler', () => require('scheduler/unstable_mock'))
 
 import * as React from 'react'
+import * as THREE from 'three'
 import { render, RenderResult } from '@testing-library/react'
 import { createWebGLContext } from '@react-three/test-renderer/src/createWebGLContext'
 
-import { Canvas, act, useThree } from '../../src/web'
+import { extend, Canvas, act } from '../../src'
+
+extend(THREE)
```

Comment:
Maybe we should do this automatically for tests in setupTests?

Reply:
I probably would yes.